src/main/scala/io/iohk/ethereum/ledger/BlockExecution.scala

```diff
@@ -81,6 +73,18 @@ class BlockExecution(
     } yield execResult.copy(worldState = worldPersisted)
   }
 
+  protected def buildInitialWorld(block: Block, parentHeader: BlockHeader): InMemoryWorldStateProxy = {
```

Comment:
ok this is weird. apparently this simple change causes this error : 
```
[error] /.../mantis/src/main/scala/io/iohk/ethereum/ledger/BlockExecution.scala:146:3: Could not find a
ny member to link for "BlockExecutionError".
[error]   /** Executes and validates a list of blocks, storing the results in the blockchain.
[error]   ^
[error] one error found
[error] Scaladoc generation failed
```
The class is mentioned in the scaladoc, and replacing it with a full name with the namespace solves the problem. But I don't really understand how this change causes a problem.